README.md

```diff
@@ -103,105 +113,170 @@ The user microservice manages the lifecycle of users and user login within the A
 
-### Notification Microservice v1_1:
+	gradle clean build
+	gradle libertyStart
 
+### Notification Microservice v1_1:
 The notification microservice v1_1 is a notification service that logs to a file for administrative purposes and sends twitter notifications to those users associated to the occasion. To receive twitter notifications, users must have a valid twitter handle in their user profile.
 
 #### Configuration:
-
 * To allow users to receive tweets, complete the [Twitter Configuration](#twitter-configuration) (This may have already been done when configuring the user microservice)
 
-* Optionally, update the location/name of the administrative log and fallback log by updating the entry located under the
-   "Optional Microservice Configuration" label in the "properties" section of the
-   sample-acmegifts/microservice-notification_v1_1 pom.xml. The default values are: 
+* Optionally, Update the location/name of the administrative log and fallback log.
+##### Maven
+Update the entry located under the "Optional Microservice Configuration" label in the "properties" section of the sample-acmegifts/microservice-notification_v1_1 pom.xml. The default values are: 
    
    `notification_1_1.log.file = ${project.build.directory}/logs/${project.build.directory}/logs/notifications_1_1.log`
    
    `notification_1_1.fallback.log.file = ${project.build.directory}/logs/notifications_1_1_fallback.log`   
+   
+##### Gradle
+Update the entries defined in the ExtraPropertiesExtension (`ext`). The default values are:
+   
+   `notification_v1_1_logfile = new File("$buildDir/logs/notifications_1_1.log")`
+   
+   `notification_v1_1_fallback_logfile = new File("$buildDir/logs/notifications_1_1_fallback.log")`
 
 #### Build and Start
+##### Maven
 Issue the following commands from within sample-acmegifts/microservice-notification_v1_1 directory
 
     mvn clean package install
     mvn liberty:start-server
+	
+##### Gradle
+Issue the following commands from within sample-acemgifts/microservice-notification_v1_1 directory
+
+	gradle clean build
+	gradle libertyStart
 
 ### Frontend UI:
 
-The Acme Gifts UI is written in Angular 2 and built using the frontend-maven-plugin. This plugin downloads and installs local versions of Node and NPM which makes calls to Angular CLI to install and build the source code. Compiled files are placed in src/main/webapp which Maven builds into a war to deploy in Liberty.
+The Acme Gifts UI is written in Angular 2 and built in Maven using the frontend-maven-plugin. This plugin downloads and installs local versions of Node and NPM which makes calls to Angular CLI to install and build the source code. Gradle uses [ysb33r's Gradle nodejs plugin](https://gitlab.com/ysb33rOrg/nodejs-gradle-plugin) to similarly install local versions of Node and NPM to build the source code. Compiled files are placed in src/main/webapp which Maven builds into a war to deploy in Liberty.
```

Comment:
Compiled files are placed in src/main/webapp which Maven builds into a war to deploy in Liberty.
Is that true for Gradle as well?

Reply:
yes. i'll correct that sentence to reflect Gradle too